app/login/page.tsx

```diff
@@ -3,24 +3,22 @@ import { useState, ChangeEvent, useEffect } from 'react';
 import Link from 'next/link';
 import { useRouter } from 'next/navigation';
 import Logo from '@/components/Logo/Logo';
-import logo from '/public/assets/logo-colored-outline.svg';
+import logo from '@/public/assets/logo-colored-outline.svg';
 import { Input } from '@/components/Input/Input';
 import { Button } from '@/components/Button/Button';
-import { loginAccount } from '@/api/apiFunctions';
+import { useAuthContext } from '@/context/AuthContextProvider';
 
-export default function Login({
-  searchParams,
-}: {
-  searchParams: { message: string };
-}) {
+export default function Login() {
   const router = useRouter();
   const [email, setEmail] = useState<string>('');
   const [password, setPassword] = useState<string>('');
-  let session: any;
+  const { loginAccount, isSignedIn } = useAuthContext();
 
-  if (typeof window !== 'undefined') {
-    session = window.localStorage.getItem('cookieFallback');
-  }
+  useEffect(() => {
+    if (isSignedIn) {
+      router.push('/weeklyPicks');
+    }
+  }, [isSignedIn]);
```

Comment:
What happens when we remove isSignedIn here in the useEffect dependency array?

Reply:
great question .. this basically keeps the page from re-rendering unless isSigned in changes.  Rather than an empty array the runs on every page load.